installer/modules_test.go

```diff
@@ -512,15 +516,17 @@ func TestStreamsTreeClearsProvidesAttribute(t *testing.T) {
 		CompatibleDevices: []string{"test-device"},
 	}
 
+	updateType := "test-type"
+
 	headerInfo := artifact.NewHeaderInfoV3([]artifact.UpdateType{
 		artifact.UpdateType{
-			Type: "test-type",
```

Comment:
Wow, that was annoying! And I guess if this was a proper lib version, it'd actually be breaking